tests/algorithms/sbi/test_abc_run_methods.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+import torch
+
+import sbibm
+from sbibm.algorithms.sbi.mcabc import run as run_mcabc
+from sbibm.algorithms.sbi.smcabc import run as run_smcabc
+from sbibm.metrics.c2st import c2st
+
+
+@pytest.mark.parametrize(
+    "task_name",
+    ["gaussian_linear", "two_moons"],
```

Comment:
we probably get it in one line if you remove the comma at the end.

Reply:
Just for you, @janfb 
[`f3d8ed5` (#30)](https://github.com/sbi-benchmark/sbibm/pull/30/commits/f3d8ed57292748d5b3b7fee7c22c2dc2624c2f0b)